stock_move_line_qty_done_constraint/models/stock_picking_type.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPickingType(models.Model):
+
+    _inherit = "stock.picking.type"
+
+    qty_done_constraint = fields.Boolean(
```

Comment:
@rousseldenis The name is meaningless... Why not `disallow_reserved_qty_overflows`?